include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3616,48 +3619,65 @@ struct OrtApi {
+   *    an id of 64 will be inferred as the last processor of the 1st group, while 65 will be interpreted as the 1st processor of the second group.
+   *    Hence 64-65 is an invalid configuration, because a windows thread cannot be attached to processors across group boundary.
+   *
+   *  \since Version 1.14
+   */
   ORT_API2_STATUS(SetGlobalIntraOpThreadAffinity, _Inout_ OrtThreadingOptions* tp_options, const char* affinity_string);
 
   /** \brief Register custom ops from a shared library.
-  *
-  * Loads a shared library (.dll on windows, .so on linux, etc) named 'library_name' and looks for this entry point:
-  *		OrtStatus* RegisterCustomOps(OrtSessionOptions * options, const OrtApiBase* api);
-  * It then passes in the provided session options to this function along with the api base.
-  *
-  * The handle to the loaded library is automatically released by ORT when the last OrtSession that references the
-  * library handle is released. If no OrtSession is created, then the library handle is released when the provided
-  * OrtSessionOptions is released.
-  *
-  * \param[in] options The session options.
-  * \param[in] library_name The name of the shared library to load and register. Refer to OS-specific dynamic library
-  *                         loading utilities (e.g., LoadLibraryEx on Windows or dlopen on Linux/MacOS) for information
-  *                         on the format of library names and search paths.
-  *
-  * \snippet{doc} snippets.dox OrtStatus Return Value
-  */
+   *
+   * Loads a shared library (.dll on windows, .so on linux, etc) named 'library_name' and looks for this entry point:
+   *		OrtStatus* RegisterCustomOps(OrtSessionOptions * options, const OrtApiBase* api);
+   * It then passes in the provided session options to this function along with the api base.
+   *
+   * The handle to the loaded library is automatically released by ORT when the last OrtSession that references the
+   * library handle is released. If no OrtSession is created, then the library handle is released when the provided
+   * OrtSessionOptions is released.
+   *
+   * \param[in] options The session options.
+   * \param[in] library_name The name of the shared library to load and register. Refer to OS-specific dynamic library
+   *                         loading utilities (e.g., LoadLibraryEx on Windows or dlopen on Linux/MacOS) for information
+   *                         on the format of library names and search paths.
+   *
+   * \snippet{doc} snippets.dox OrtStatus Return Value
+   */
   ORT_API2_STATUS(RegisterCustomOpsLibrary_V2, _Inout_ OrtSessionOptions* options, _In_ const ORTCHAR_T* library_name);
 
+  /** \brief Register custom ops by calling a RegisterCustomOpsFn function.
+   *
+   * Searches for registration_func_name and if found calls it. The library containing the function must either be
```

Comment:
Is it worthwhile to mention as part of API docs why a user might prefer this to other RegisterCustomOps* functions?

Reply:
Added a bit more info. Let me know if there's anything else required.